### ISSUE TITLE
SSSOM test fails because of undeclared prefixes in RO import

### ISSUE THREAD

**Issue by gouttegd** [INCATools/ontology-development-kit#809]
The test suite currently fails at the `test-sssom.yaml` test:

```
sssom parse tmp/modmini-ontc.obographs.json -I obographs-json  -o ../mappings/modmini-ontc.sssom.tsv
WARNING:root:Wikipedia is used in the SSSOM mapping set but it does not exist in the prefix map
WARNING:root:Reactome is used in the SSSOM mapping set but it does not exist in the prefix map
Traceback (most recent call last):
  File "/usr/local/bin/sssom", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.10/dist-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/sssom/cli.py", line 210, in parse
    parse_file(
  File "/usr/local/lib/python3.10/dist-packages/sssom/io.py", line 112, in parse_file
    doc.clean_prefix_map()
  File "/usr/local/lib/python3.10/dist-packages/sssom/util.py", line 170, in clean_prefix_map
    raise ValueError(
ValueError: ['Wikipedia', 'Reactome'] are used in the SSSOM mapping set but it does not exist in the prefix map
make: *** [Makefile:456: ../mappings/modmini-ontc.sssom.tsv] Error 1
```

This is because the _ro_ import module contains cross-references to Wikipedia and Reactome. Those cross-references end up in the JSON version of the test ontology from which `sssom` later tries to extract the `modmini-ontc.sssom.tsv` mapping set. `sssom parse` does not know what to do with the `Wikipedia:` and `Reactome:` prefixes and errors out as shown above.

Possible solutions:

A) Update the generated SSSOM workflow to add the `--no-clean-prefixes` option to the `sssom parse` call, to silently ignore cross-references with unknown prefixes.
B) Update the test to make it use a custom Makefile with an overloaded rule to generate the mapping file, with either the `--no-clean-prefixes` option or a custom prefix map.
C) Disable the test altogether until a better solution is found.
D) Don’t use RO as an import module in this test; find another ontology to use, that doesn’t contain undeclared prefixes – and hope that it won’t change.

But ultimately, the _real_ solution is:

E) We should stop using _live_, real-world data in the test suite. If we really want to use real-world ontologies, then we should take a frozen snapshot of these ontologies at a given point in time, store those snapshots in this very repository, and only use the snapshots in the test suite.

Using live ontologies freshly downloaded from their sources means the test suite can fail anytime because of a random change _outside_ of the ODK. This is certifiably insane.

**Comment by matentzn** [comment]
I think we could and should do D) in combination with a version IRI.. Maybe just COB.

**Comment by matentzn** [comment]
This has been fixed bu #822
